rootston/desktop.c

```diff
@@ -817,6 +803,15 @@ struct wlr_surface *desktop_surface_at(struct roots_desktop *desktop,
 					ox, oy, sx, sy))) {
 			return surface;
 		}
+
+		struct roots_output *output =
+			desktop_output_from_wlr_output(desktop, wlr_output);
+		if (output != NULL && output->fullscreen_view != NULL) {
+			if (view_at(output->fullscreen_view, lx, ly, &surface, sx, sy)) {
+				return surface;
+			}
```

Comment:
If there's a fullscreen view, we don't want to check for views/layers underneath.

Reply:
Ah, makes sense. Updated!